bpm/bonita-core/bonita-process-definition/bonita-process-definition-impl/src/main/java/org/bonitasoft/engine/core/process/definition/ProcessDefinitionServiceImpl.java

```diff
@@ -176,14 +182,22 @@ public void disableProcessDeploymentInfo(final long processId) throws SProcessDe
                     .setObject(processDefinitionDeployInfo).done();
         }
         try {
-            recorder.recordUpdate(updateRecord, updateEvent);
+            update(processId, processDefinitionDeployInfo, updateRecord, updateEvent);
             log(processDefinitionDeployInfo.getId(), SQueriableLog.STATUS_OK, logBuilder, "disableProcess");
-        } catch (final SRecorderException e) {
+        } catch (final SRecorderException | SCacheException e) {
             log(processDefinitionDeployInfo.getId(), SQueriableLog.STATUS_FAIL, logBuilder, "disableProcess");
             throw new SProcessDisablementException(e);
         }
     }
 
+    void updateLastUpdateDateInCache(long processId, SProcessDefinitionDeployInfo processDefinitionDeployInfo) throws SCacheException {
+        final Pair<Long, SProcessDefinition> fromCache = getFromCache(processId);
+        if (fromCache != null) {
+            fromCache.setKey(processDefinitionDeployInfo.getLastUpdateDate());
+            cacheService.store(PROCESS_CACHE_NAME, processId, fromCache);
```

Comment:
Ok, got it.
On another hand, **getFromCache()** is a wrapping method, and **cacheService.store()** is lower-level method called directly, it seems not homogeneous.